### DISCUSSION TITLE
How to manipulate the camera by layout?

### DISCUSSION THREAD

**Discussion by mackymangampo** [fluttercandies/flutter_wechat_camera_picker#203] [Q&A]
For the context I'm using `CameraPickerConfig(foregroundBuilder: (context, controller) {// the layout} )` So this the output that I've changed. I just want to know how I will manipulate more the camera layout so that I can satisfy my screens. 

![image](https://github.com/fluttercandies/flutter_wechat_camera_picker/assets/44796954/d9c4536c-4218-440f-a3d4-b4c10af029c0)

This screen I need to make: **Is this possible to make camera layout?** 

<img width="562" alt="image" src="https://github.com/fluttercandies/flutter_wechat_camera_picker/assets/44796954/26f3bd66-bf66-4374-84d7-f4ee009c9cf5">

**Comment by AlexV525** [comment]
Extends `CameraPickerState` should resolve your demand.

**Reply by mackymangampo** [reply]
Can you give me details or example so that I have an reference on how do Implement it?

**Reply by AlexV525** [reply]
https://github.com/fluttercandies/flutter_wechat_camera_picker?tab=readme-ov-file#using-custom-states We only have a simple usage in the doc for now. Check the state's source code to determine which part of the layout you want to override.

**Reply by mackymangampo** [reply]
I just reviewing package docs, I can't see the part where I can override the function when confirming the image, 
So that I can override the layout of this feature is it possible?

See the image for reference:
**This part**

![image](https://github.com/fluttercandies/flutter_wechat_camera_picker/assets/44796954/96b7b76d-79c8-48c0-9eeb-120cf6e83ff6)

**Reply by AlexV525** [reply]
![image](https://github.com/fluttercandies/flutter_wechat_camera_picker/assets/15884415/52f8a7d9-628b-4b28-9fc5-68e828c2d69a)
